agent/terraform/main.tf

```diff
@@ -0,0 +1,57 @@
+
+variable "juju_model" {
```

Comment:
Even though this is small
it's probably worth following the normal terraform structure where you put 
variables in variables.tf
resources in main.tf
outputs in outputs.tf
and providers in versions.tf

it isnt that the current structure is wrong, but its good to move toward consistency.

Reply:
Good idea, updated!